app/forms/validators.py

```diff
@@ -183,6 +185,7 @@ def __call__(
             field.raw_data[0]
             .replace(numbers.get_group_symbol(flask_babel.get_locale()), "")
             .replace(" ", "")
+            .replace("_", "")
```

Comment:
We seem to have an inconsistent amount of call like these, I suggest make a helper method that, does this given an input, replace group symbol, whitespace and `_` then return the value. Then we only have a single place to update, this is done in like 4 places now.